modules/ui/src/main/scala/lucuma/ui/sequence/SequenceRowBuilder.scala

```diff
@@ -76,7 +76,11 @@ trait SequenceRowBuilder[D]:
   protected def renderCurrentHeader(sequenceType: SequenceType): VdomNode =
     <.span(SequenceStyles.CurrentHeader, sequenceType.toString)
 
-  protected def renderVisitExtraRow(step: SequenceRow.Executed.ExecutedStep[D]) =
+  protected def renderVisitExtraRow(
```

Comment:
Would we be able to set QA from either observe and explore?

Reply:
This provides UI support, which has a common renderer in both observe and explore. Currently, plans are only to implement this from observe, but nothings stops us from doing it in explore too.